lib/events/stream.go

```diff
@@ -161,6 +161,12 @@ func (s *ProtoStreamer) ResumeAuditStream(ctx context.Context, sid session.ID, u
 	})
 }
 
+// GetUploadMetadata gets the session upload metadata
```

Comment:
I realized that we don't need the streamer to implement `UploadMetadataGetter` - the `UploadHandler` alone is enough.